pkg/helpers/mock_interfaces.go

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
```

Comment:
Packages like "util" and "helpers" are generally to be avoided where possible. Generally mocks should be colocated with the packages that they are mocking which should remove the need for this pkg.

Reply:
I'll move it to core